mne/time_frequency/tfr.py

```diff
@@ -333,10 +351,11 @@ def _time_frequency(X, Ws, use_fft):
         tfrs = _cwt_convolve(X, Ws, mode)
 
     for tfr in tfrs:
-        tfr_abs = np.abs(tfr)
-        psd += tfr_abs ** 2
```

Comment:
I think this was really the underlying offending line. This makes the order of operations decompose, square, average instead of decompose, average, square.